README.md

```diff
@@ -126,3 +126,21 @@ We recommend your smartphone be connected to the same WiFi network as your compu
 
 - [ ] https://reactnative.dev/docs/tutorial
 - [ ] https://reactnative.dev/docs/getting-started
+
+
+## Contributions
```

Comment:
We shouldn't be changing the README.md at the root of the project.

Reply:
I'll go back and correct it - when that happens, would I make a whole new pull request?
Similarly, if we work on multiple issues, do you do one pull request for all of them?

Reply:
@martinsliz you don't have to create a new pull request. When you get feedback, you just go back to your code, in the branch used for the pull request, and add more commits, then push. GitHub will automatically update the pull request page.

1. Find an issue to work on
2. Create a feature branch locally:
```
git checkout master
git checkout -b my-feature-branch
# ... do your work ... (commits)
git push origin my-feature-branch
```
3. Create a pull request on GitHub that will take your fork's my-feature-branch and merge into the original repository's master branch.
4. Reviewer will leave feedback
5. Go back to your code locally, in the same branch my-feature-branch, and add & commit further changes.

(Note for this pull request you didnt create a feature branch, but I'll let this one pass)

If you work on multiple issues, try to make a feature branch for each issue (remember to first checkout master so you always branch off of master, not your feature branch), with a different pull request for each.

Q: Are there times where it's OK to do multiple issues in one pull request?
A: Yes, if the features are very closely related to each other, it's OK to do all the work in one single feature branch and a single pull request.

However for the sake of practice in this bootcamp, I recommend one issue, one feature branch, one pull request. So that you can practice the steps. Repetition gives you experience.

Reply:
I did indeed forget to make a branch, so yes it makes total sense to do one at a time here for the muscle memory. Thank you for all the feedback!